src/model/transformations/redirection-filters.js

```diff
@@ -17,7 +17,7 @@ function getFiltersState(filter, parent, type) {
   );
   return {
     id,
-    name,
+    name: name || false,
```

Comment:
Why not a empty string instead of setting a boolean value to a string variable ?